test/runtime_tests.cc

```diff
@@ -1013,5 +1015,50 @@ namespace detail {
 		dry_run_with_nodes(nodes);
 	}
 
+	TEST_CASE_METHOD(test_utils::mpi_fixture, "Config reads correctly environment varibles", "[env-vars]") {
```

Comment:
The test name is not proper english. Should probably be something like `"Config reads environment variables correctly"`.